analyticsdataserver/tests/test_utils.py

```diff
@@ -0,0 +1,22 @@
+import logging
```

Comment:
This should be the same as the tests that were in `analyticsdataserver/tests.py`.